packages/kbn-rule-data-utils/src/default_alerts_as_data.ts

```diff
@@ -67,6 +67,9 @@ const ALERT_UUID = `${ALERT_NAMESPACE}.uuid` as const;
 // kibana.alert.workflow_status - open/closed status of alert
 const ALERT_WORKFLOW_STATUS = `${ALERT_NAMESPACE}.workflow_status` as const;
 
+// kibana.alert.workflow_tags - user workflow alert tags
+const ALERT_WORKFLOW_TAGS = `${ALERT_NAMESPACE}.workflow_tags` as const;
```

Comment:
@elastic/actionable-observability we chose to create a new field here instead of using an existing one (`tags` was our original choice) because we found that, since we copy over source event tags into the top level `tags` field, it was possible to "merge" these event tags with our user based tags potentially leading to some confusing workflows or breaking them all together. We went with `workflow_tags` as the usage for the tags in this feature aligned fairly well with the existing other workflow fields, and we wouldn't have to worry about breaking existing workflows or effectively having source event data be mutable by using `tags`